test/e2e/policyrecommendation_test.go

```diff
@@ -115,23 +175,89 @@ func testPolicyRecommendationStatus(t *testing.T, data *TestData) {
 	assert.Containsf(stdout, "Status of this policy recommendation job is", "stdout: %s", stdout)
 }
 
+// Example output:
+// Status of this policy recommendation job is Failed
+// Error message: driver pod not found
```

Comment:
`driver pod not found` is the error msg exported by spark so I can't change it to `Pod`
I capitalized `Error message` because it starts a new line. For the error msg inside `fmt.Errorf` and returned, we usually avoid capitalizing it.

Reply:
> driver pod not found is the error msg exported by spark so I can't change it to Pod

got it

> I capitalized Error message because it starts a new line. For the error msg inside fmt.Errorf and returned, we usually avoid capitalizing it.

no worry, I didn't make myself clear. I was trying to ask is Pod intentionally written into pod